.github/workflows/build_and_push_to_docker.yml

```diff
@@ -30,23 +72,129 @@ jobs:
         password: ${{ secrets.DOCKER_PASSWORD }}
 
     - uses: actions/checkout@v2
-    - name: Build renku project docker images
+    - name: Build renku project python-based docker images
```

Comment:
```suggestion
    - name: Build renku project python-based extension images
```

Maybe?

Reply:
good point; changed to: `Build renku project python-based docker image extensions`